src/reference/supported-platforms.md

```diff
@@ -4,52 +4,78 @@ short-title: Supported platforms
 description: The platforms that Flutter supports by platform version.
 ---
 
-## Support tiers
-
-We define three tiers of support for the platforms on
-which apps built with Flutter might be deployed:
-
-1. **Supported**<br>
-   Google-tested platforms that
-   are automatically tested on every commit
-   by continuous integration testing.
-1. **Best effort**<br>
-   Platforms that we intend to support through
-   coding practices,
-   but are only tested on an ad-hoc basis.
-1. **Unsupported**<br>
-   Platforms that we don't test or support.
-   
-{% comment %}
-**IMPORTANT NOTE**:
-When changing the minimum support version of macOS, Windows, or Linux,
-make sure to make a corresponding change in the get started pages:
-  * /src/get-started/install/macos.md
-  * /src/get-started/install/windows.md
-  * /src/get-started/install/linux.md
-{% endcomment -%}
-
-
-## Deploying Flutter
-
-As of the current stable release,
-support for deploying Flutter apps is shown in the
-following table:
+As of Flutter {{site.appnow.flutter}},
+Flutter supports the following hardware architectures and platform versions
+to run Flutter code.
+
+## Supported hardware architectures
+
+Flutter supports running apps on the following hardware architectures.
+
+{% assign chips = site.data.chipsets %}
+{% capture yes %}
+ <span class="material-symbols" 
+       style="color: #158477"
+       aria-label="The Flutter SDK supports the specified architecture on the specified target platform"
+       role="img">verified</span>
+{% endcapture %}
+{% capture no %}
+ <span class="material-symbols"
+       style="color: #D43324"
+       aria-label="The Flutter SDK does not support the specified
+       architecture on the specified target platform"
+       role="img">dangerous</span>
+{% endcapture %}
+{% capture beta %}
+ <span class="material-symbols"
+       style="color: #13C2AD"
+       aria-label="The Flutter SDK supports ARM64 in beta the only architecture on Windows"
+       role="img">gpp_maybe</span>
+{% endcapture %}
+{% capture na %}
+ <span class="material-symbols"
+       style="color: #DADCE0"
+       aria-label="No version of the Flutter SDK exists for the
+       specified architecture on the specified target platform"
+       role="img">do_not_disturb_on</span>
+{% endcapture %}
+
+| Platform |   x86   |   x64   |  ARM32  |   ARM64   | RV64GC   |
+|---|:---:|:---:|:---:|:---:|:---:|
+{%- for chip in chips %}
+  | {{chip.platform}} | {{chip.x86}} | {{chip.x64}} | {{chip.arm32}} | {{chip.arm64}} | {{chip.risc}} |
+{%- endfor %}
+{:.table.table-striped}
+
+<div aria-hidden="true" markdown="1">
+
+{{yes}} Supported in all channels.  
+{{no}} Unsupported in all channels.  
+{{beta}} Supported in Beta channel only.  
+{{na}} No version exists.  
```

Comment:
- For more reliable and easier to notice forced new-lines, you can use `<br>` here. We do it elsewhere on the sites as well.
- Generally I think of an OS or architecture as "supported on" or "supported by".
- To highlight channels, we generally use code font or bold while keeping it lower case.
- "No version exists" could be mistaken to mean that no version exists for Flutter. I think it would be beneficial to expand a bit to different it from the case where it's just unsupported.

```suggestion
{{yes}} Supported on all channels.<br>
{{no}} Unsupported on all channels.<br>
{{beta}} Supported on the `beta` and `main` channels only.<br>
{{na}} Unsupported by the operating system.<br>
```

Reply:
@parlough : No issue with most of this, but the N/A comes from mit's thoughts on Dart. I don't know if I want to confuse this if you look across both sites.

Reply:
I would also like to keep markup to Markdown wherever possible.